marshmallow_jsonapi/fields.py

```diff
@@ -63,20 +76,42 @@ def __init__(
         self,
         related_url='', related_url_kwargs=None,
         self_url='', self_url_kwargs=None,
-        include_data=False, many=False, type_=None, id_field=None, **kwargs
+        include_resource_linkage=False, schema=None,
+        many=False, type_=None, id_field=None, **kwargs
     ):
         self.related_url = related_url
         self.related_url_kwargs = related_url_kwargs or {}
         self.self_url = self_url
         self.self_url_kwargs = self_url_kwargs or {}
-        if include_data and not type_:
-            raise ValueError('include_data=True requires the type_ argument.')
+        if include_resource_linkage and not type_:
+            raise ValueError('include_resource_linkage=True requires the type_ argument.')
         self.many = many
-        self.include_data = include_data
+        self.include_resource_linkage = include_resource_linkage
+        self.include_data = False
+        self.__schema = schema
         self.type_ = type_
         self.id_field = id_field or self.id_field
         super(Relationship, self).__init__(**kwargs)
 
+    @property
+    def schema(self):
+        if isinstance(self.__schema, SchemaABC):
+            return self.__schema
+        if isinstance(self.__schema, type) and issubclass(self.__schema, SchemaABC):
+            self.__schema = self.__schema(many=self.many)
```

Comment:
@Tim-Erwin Have you tested with `self.many=True`?   I think this line should be changed to have `many=False`.
```python
self.__schema = self.__schema()
```
The code currently iterates over the relationship's data on [line 196](https://github.com/marshmallow-code/marshmallow-jsonapi/pull/40/files#diff-b882a3f5a63ddce7dd43a950cd1d20d1R198) and dumps each item separately.

Reply:
Is this why it breaks when trying to include data that is ```many=False``` when the root object is ```many=True```?